Wazuh Cloud: Documentation for the new pricing model (#6258)
wazuh/wazuh-documentation: source/cloud-service/getting-started/starting-faq.rst

  
- `Is it PCI DDS compliant?`_

- `Is it SOC 2 compliant?`_

- `How can I get support?`_

- `Where is Wazuh Cloud hosted?`_

- `What is a profile?`_

- `What is a setting?`_

- `What is the indexed data?`_

- `What is the archived data?`_

- `What is a tier?`_

- `What happens if the tier limit is reached?`_

- `What happens if the indexed data size setting is reached?`_

- `How is indexed data rotated?`_

- `What happens if the data ingestion rate is exceeded?`_

- `Can I cancel at any time?`_
  
What is Wazuh Cloud?
--------------------

> Wazuh Cloud provides a comprehensive solution by hosting and managing all Wazuh components within a single integrated platform. With just a few clicks, you can effortlessly create your environment and enroll Wazuh agents. By leveraging Wazuh, you gain access to powerful security capabilities, including Security Information and Event Management (SIEM) and Extended Detection and Response (XDR).

Reviewer: ```suggestion
Wazuh Cloud provides a comprehensive solution by hosting and managing all Wazuh components within a single integrated platform. Wazuh cloud allows you to create your environment and enroll Wazuh agents. By leveraging Wazuh, you gain access to security capabilities, including Security Information and Event Management (SIEM) and Extended Detection and Response (XDR).
```

Can I try it for free?
----------------------

Yes, Wazuh provides a free trial to create a cloud environment and access the Wazuh Cloud service. You can :ref:`sign up for a 14-day free trial <cloud_sign_up>`, and no credit card information is required to complete this process.

Will I be charged when my trial is over?
----------------------------------------

No, Wazuh does not request credit card information to start your trial. Charges are only applied if you keep using the service beyond the trial period.

Is it PCI DDS compliant?
------------------------

Yes, the Wazuh Cloud platform is validated as PCI DSS Level 1 Service Provider compliant.

Is it SOC 2 compliant?
----------------------

Yes, the Wazuh Cloud platform is compliant with SOC 2 standards.

.. _cloud_getting-started_support:

How can I get support?
----------------------

> Every environment includes support after the first payment. You can contact the Wazuh team anytime through the Help section on your Wazuh Cloud Console. You can also fill out `this form <https://wazuh.com/cloud/#subscription>`_ to get help from the Wazuh team.

Reviewer: ```suggestion
Every environment includes support after the initial payment. You can contact the Wazuh team anytime through the Help section on your Wazuh Cloud Console. You can also fill out `this form <https://wazuh.com/cloud/#subscription>`_ to get help from the Wazuh team.
```

Where is Wazuh Cloud hosted?
----------------------------

Wazuh Cloud is hosted on Amazon Web Services (AWS).

What is a profile?
------------------

> A profile refers to predefined settings that you can choose from when configuring your Wazuh Cloud environment. We have three profiles available: Small, Medium, and Large. These profiles provide ready-to-use environment templates that cater to different needs and requirements. If none of the predefined profiles meet your specific requirements, you can configure your settings individually.

Reviewer: ```suggestion
A profile refers to predefined settings available for configuring your Wazuh Cloud environment. We offer three profiles: Small, Medium, and Large. They provide ready-to-use environment templates that cater to different needs and requirements. If none of these predefined profiles meet your specific requirements, you can configure your settings individually.
```

What is a setting?
------------------

> A setting in the context of Wazuh Cloud refers to each configuration option available for a cloud environment. These settings determine the limitations and capabilities of an environment. For example, the "Active Agents" setting specifies the maximum number of active agents allowed in your environment. It is important to note that the settings chosen for an environment directly impact its pricing, as they affect the resources allocated and functionalities available.

Reviewer: ```suggestion
In the context of Wazuh Cloud, a setting  refers to each configuration option available for a cloud environment. These settings determine the limitations and functionalities of an environment. For example, the "Active Agents" setting specifies the maximum number of active agents allowed in your environment. Note that the settings chosen for an environment directly impact its pricing, as they affect the resources allocated and functionalities available.
```

What is the indexed data?
-------------------------

> The indexed data, previously known as hot storage, includes the data available on the Wazuh WUI, which corresponds to the information indexed by Wazuh. This information becomes searchable and analyzable as soon as Wazuh ingests and indexes the events sent by the agents.

Reviewer: ```suggestion
The indexed data, previously known as hot storage, includes the data available on the Wazuh dashboard, which corresponds to the information Wazuh indexed. This information becomes searchable and analyzable as soon as Wazuh ingests and indexes the events sent by the agents.
```

What is the archived data?
--------------------------

> This Wazuh generated data is archived in an AWS S3 bucket for long-term storage purposes. Unlike the indexed data, this data is not searchable or analyzable. It simply consists of a collection of compressed files. For more information, please see the :ref:`Archived data <cloud_your_environment_accessing_archived_data>` section.

Reviewer: ```suggestion
The data generated by Wazuh is archived in an AWS S3 bucket for long-term storage purposes. Unlike the indexed data, this archived data is not searchable or analyzable. It simply consists of a collection of compressed files. For more information, please see the :ref:`Archived data <cloud_your_environment_accessing_archived_data>` section.
```

What is a tier?
---------------

> The concept of a tier, which represents the size limitation, in bytes, of the indexed data (formerly known as hot storage), is no longer used. It has been replaced by the *indexed data size* setting.

Reviewer: ```suggestion
The term *tier* is no longer used. It has been replaced by the *indexed data size* setting. It represented the size limitation, in bytes, of the indexed data, formerly known as *hot storage*.
```

What happens if the tier limit is reached?
------------------------------------------

See :ref:`What happens if the indexed data size setting is reached? <cloud_starting_faq_data_reached>`.

.. _cloud_starting_faq_data_reached:

What happens if the indexed data size setting is reached?
---------------------------------------------------------

> When the selected **indexed data size** is reached, the oldest events will be automatically removed from your index regardless of the **index data time**. This data is available in archived data for you to access. See the :ref:`Archived data <cloud_your_environment_accessing_archived_data>` section to learn more about data logging and storage.

Reviewer: ```suggestion
When the selected **indexed data size** is reached, Wazuh automatically removes the oldest events from your index, regardless of the **index data time**. You can access this data in archived data. See the :ref:`Archived data <cloud_your_environment_accessing_archived_data>` section to learn more about data logging and storage.
```

How is indexed data rotated?
----------------------------

> Data rotation is determined by two conditions: the indexed data time and the indexed data size. For instance, if you have set a 3-month indexed data time and a 100GB indexed data size, and you consume the entire 100GB within the first month, the data will start rotating. Similarly, even if you have not utilized all 100GB, the data from the first month will be rotated by the fourth month if only 20GB is used.

Reviewer: ```suggestion
Data rotation is determined by two conditions:

-  Indexed data time
-  Indexed data size

For example, you might have set a 3-month indexed data time and a 100 GB indexed data size. If you consumed the entire 100 GB within the first month, the data starts rotating.

Similarly, you might have not utilized all 100 GB. However, the data from the first month rotates by the fourth month even if only 20 GB is used.
```

What happens if the data ingestion rate is exceeded?
----------------------------------------------------

> If you send more events per second than allowed by the **data ingestion rate** setting, events start queuing, and if the queue becomes full, the incoming events will be discarded, which may lead to potential event loss.

Reviewer: ```suggestion
If you send more events per second than allowed by the **data ingestion rate** setting, events start queuing, and if the queue becomes full, the incoming events get discarded, which might lead to potential event loss.
```
